CHANGELOG.md

```diff
@@ -20,6 +20,15 @@ Client Library | v0.10.1  | [symbol-openapi-typescript-fetch-client](https://www
 - Added `transferMosaicId`, `fromTransferAmount`, `toTransferAmount` to transaction searches.
 - Added `CurrencyService` to allow loading Network and custom `Currency` objects from the rest service.
 - Added `UnlockedAccount` endpoint in `NodeRepository` for checking delegated harvesting status.
+- Added `StateProofService` to verify the different states.
+- Added `serialize()` to state objects like `AccountInfo`, `MosaicInfo`, etc.
+- Added `version` field to state objects.
+- Added `/merkle` endpoints to the repositories of the different states.
+- Added `stremer()` to repositories to simplify `PaginationStreamer` objects creation.
+- Improved `search` endpoints allowing "empty" criteria in order to paginate over all the objects.
+- `Listener` now accepts address aliases as `UnresolvedAddress` objects.
+- Added V1 and V2 Voting Key transaction support.
+- Updated `FinalizationProof` objects to the latest server changes.
```

Comment:
Updated `FinalizationProof` object added `SignatureSchema` for server tree `testnet/v3`